### PR TITLE
Make PR template more pertinent to docs changes

### LINKED ISSUE #9: Create an issue / PR template that makes sense for documentation
This should loosely model the templates used for the "faas" repo but have specific questions / prompts which relate to adding documentation.

Checkpoints may include:

* [ ] I have tested the markdown appearance with Docker
* [ ] I have tested any new instructions
* [ ] I am the author or have permission to share this text and reference the source

### PR DESCRIPTION
**Pull request by rgee0**
Signed-off-by: Richard Gee <richard@technologee.co.uk>

## Description
 - Small change to README so that the local development section can be linked from the new template.
 - Pull in suggestions from the original issue and remove less relevant checklist items

## Motivation and Context
- [x] I have raised an issue to propose this change (fixes #9 )

## How Has This Been Tested?
Its an issue template

## Types of changes
- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:
- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I've read the [CONTRIBUTION](https://github.com/openfaas/faas/blob/master/CONTRIBUTING.md) guide
- [x] I have signed-off my commits with `git commit -s`

### CODE REVIEW

**Comment by jonatasbaldin** [comment]
Nice, I like it 💃

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -10,21 +10,24 @@
...
+- [ ] Non-content (config/templates/mkdocs/CSS)
 
 ## Checklist:
 <!--- Go over all the following points, and put an `x` in all the boxes that apply. -->
-<!--- If you're unsure about any of these, don't hesitate to ask. We're here to help! -->
 - [ ] My code follows the code style of this project.
-- [ ] My change requires a change to the documentation.
-- [ ] I have updated the documentation accordingly.
+- [ ] I have tested any new instructions
+- [ ] I am the author
```
Comment (alexellis) [inline .github/PULL_REQUEST_TEMPLATE.md]: I think this declaration makes sense, but some people (international speakers, perhaps) may assume this means that they created the PR. Can we be more specific, but remain concise?

Do we need both?

What about:

`- [ ] I have included  copyrighted materials from third parties.`
`- [ ] I have permission to use copyrighted materials from third parties.`

Comment (alexellis) [reply .github/PULL_REQUEST_TEMPLATE.md]: To some extent this is also covered under the DCO

Comment (rgee0) [reply .github/PULL_REQUEST_TEMPLATE.md]: addressed

**Comment by Waterdrips** [comment]
I like this a lot!

**Pull request closed without merging**